app/Models/ContributionPlan.php

```diff
@@ -19,14 +19,14 @@ class ContributionPlan extends Model
      */
     protected $guarded = [];
 
-    /**
-     * Make the contribution_id into a custom readable format.
-     *
-     * @return string
-     */
-    public function getContributionIdAttribute() {
-        return "Loan - {$this->contribution_id}";
-    }
+//    /**
```

Comment:
@enwemasor1995 Why is this function commented out?

Reply:
It seems not to do anything and was causing error when I use custom query for get contribution plan ......I left because I wasn't totally sure